bitbucket-pipelines.yml

```diff
@@ -111,3 +111,18 @@ pipelines:
   branches:
     # Using the default_steps anchor means no other items can be added
     master: *default_steps
+  custom:
+    clu:
+    - step:
+        name: Composer Lock Update
+        services:
+          - docker
+        caches:
+          - docker
+          - composer
+        script:
+          - *bash_env_export
+          - *bash_env_source
```

Comment:
I replaced these two lines with
`          - /build-tools-ci/scripts/set-environment`
 to get the job to build in Pipelines

Reply:
@aaronbauman good catch. Since there is no prior job the Bash artifact is not preserved/available to the CLU job

Reply:
Updated in `2aa4be2`